java/KinesisConnectors/src/main/resources/flink-application-properties-dev.json

```diff
@@ -0,0 +1,17 @@
+[
+  {
+    "PropertyGroupId": "InputStream0",
+    "PropertyMap": {
+      "stream.arn": "arn:aws:kinesis:us-east-1:060795913058:stream/ExampleInputStream",
+      "kinesis.stream.init.position": "LATEST",
+      "aws.region": "us-east-1"
+    }
+  },
+  {
+    "PropertyGroupId": "OutputStream0",
+    "PropertyMap": {
+      "stream.arn": "arn:aws:kinesis:us-east-1:060795913058:stream/ExampleOutputStream",
```

Comment:
As above

Reply:
Changed `stream.arn` values to use a dummy account id `012345678900`.